services/meminator-nodejs/src/o11yday-lib.ts

```diff
@@ -5,7 +5,7 @@
 
 import { W3CTraceContextPropagator } from '@opentelemetry/core';
 import { context, defaultTextMapSetter, trace, Attributes, SpanStatusCode, Span, SpanKind, SpanOptions } from '@opentelemetry/api';
-import { SEMATTRS_HTTP_METHOD, SEMATTRS_HTTP_URL } from '@opentelemetry/semantic-conventions';
+import { ATTR_HTTP_REQUEST_METHOD, ATTR_HTTP_RESPONSE_STATUS_CODE, ATTR_URL_FULL } from '@opentelemetry/semantic-conventions';
```

Comment:
This is not actually in the course itself, just a change in another service for newer semantic conventions.